pkg/apis/nfd/v1alpha1/types.go

```diff
@@ -51,6 +51,10 @@ type NodeFeatureSpec struct {
 	// Labels is the set of node labels that are requested to be created.
 	// +optional
 	Labels map[string]string `json:"labels"`
+	// ExtendedResources is the set of extended resources that are requested to
+	// be created.
+	// +optional
+	ExtendedResources map[string]string `json:"extendedResources"`
```

Comment:
This makes ERs supported in the NodeFeature object, too. Maybe we want it but then we need to document it also everywhere (where NFs) are discussed and have separate e2e tests for it